### ISSUE TITLE
Search filter needs to be cleaned up

### ISSUE THREAD

**Issue by ekcannon** [ncgr/tripal_phylogeny#22]
When a search filter takes up the entire page and it's difficult to find the submit button, it's time to rework it... :-)

In any case, as more species and assemblies are added, the filter grows increasingly unwieldy.

**Comment by adf-ncgr** [comment]
agreed; this would be a good one for someone who is drupal-savvy, but we should probably make sure we're all agreed on what the requirements are. not only the search filter itself is growing unwieldy but also the table of results.
